app/uk/gov/hmrc/eoricommoncomponent/frontend/controllers/Sub02Controller.scala

```diff
@@ -110,8 +110,6 @@ class Sub02Controller @Inject() (
         processedDate <- sessionCache.sub01Outcome.map(
           sub01 => if (sub01.processedDate.nonEmpty) sub01.processedDate else sub02Outcome.processedDate
         )
-        _ <- sessionCache.remove
```

Comment:
Question around the complete page. If we don't remove the cache is there a risk of the application being submitted twice if the page is refreshed?